eclipse-collections/src/main/java/org/eclipse/collections/impl/bag/strategy/mutable/MutableHashingStrategyBagFactoryImpl.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2021 Bhavana Hindupur.
+ * Copyright (c) 2022 Goldman Sachs and others.
```

Comment:
This should be "2022 Bhavana Hindupur and others."

Reply:
addressed